src/main/java/com/exactpro/th2/common/schema/factory/AbstractCommonFactory.java

```diff
@@ -503,11 +502,9 @@ public InputStream readDictionary() {
      * @throws IllegalStateException if can not read dictionary
      */
     public InputStream readDictionary(DictionaryType dictionaryType) {
-
         try {
-
-            var dictionaries = Files.list(getPathToDictionariesDir())
-                    .filter(path -> Files.isRegularFile(path) && path.getFileName().toString().contains(dictionaryType.name()))
+            var dictionaries = Files.list(getPathToDictionariesDir().resolve(dictionaryType.name().toLowerCase()))
```

Comment:
We use `dictionaryType.name().toLowerCase()` in CommonFactory as well. Maybe we need to extract it to a utility function